shared/actions/json/chat2.json

```diff
@@ -0,0 +1,322 @@
+{
```

Comment:
new actions! also note this is json5 so it has comments (this was before danny's description thing)